pom.xml

```diff
@@ -0,0 +1,621 @@
+              <shadedClassifierName>SoyParseInfoGenerator</shadedClassifierName>
+              <transformers>
+                <transformer
+                  implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                  <mainClass>com.google.template.soy.SoyParseInfoGenerator
+                  </mainClass>
+                </transformer>
+              </transformers>
+            </configuration>
+          </execution>
+
+          <execution>
+            <id>SoyMsgExtractor</id>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>SoyMsgExtractor</shadedClassifierName>
+              <transformers>
+                <transformer
+                  implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                  <mainClass>com.google.template.soy.SoyMsgExtractor</mainClass>
+                </transformer>
+              </transformers>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.8</version>
+        <executions>
+          <execution>
+            <id>generated-soyutils</id>
+            <phase>package</phase>
+            <configuration>
+              <target>
+                <taskdef name="gen.escape.directives"
+                         classname="com.google.template.soy.jssrc.internal.GenerateSoyUtilsEscapingDirectiveCode"
+                         classpathref="maven.runtime.classpath"/>
+                <gen.escape.directives>
+                  <input path="${project.basedir}/javascript/soyutils_usegoog.js"/>
+                  <output path="${project.basedir}/target/soyutils_usegoog.js"/>
+                  <libdefined pattern="goog.string.*"/>
+                  <libdefined pattern="goog.asserts.*"/>
+                </gen.escape.directives>
+              </target>
+            </configuration>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>1.3.2</version>
+        <configuration>
+          <detail>true</detail>
+        </configuration>
+        <executions>
+
+          <execution>
+            <id>js-simple-example</id>
+            <phase>verify</phase>
+            <configuration>
+              <executable>java</executable>
+              <arguments>
+                <argument>-jar</argument>
+                <argument>
+                  ${project.build.outputDirectory}/../soy-${project.version}-SoyToJsSrcCompiler.jar
```

Comment:
`${project.build.directory}/soy-${project.version}-SoyToJsSrcCompiler.jar`

(or could/should we use `${project.build.finalName}` instead of `soy-${project.version}`? `${project.build.directory}/${project.build.finalName}-SoyToJsSrcCompiler.jar`)